lib/citeproc/ruby/formats/html.rb

```diff
@@ -140,7 +140,9 @@ def css
 
         def finalize_content!
           super
-          output.replace content_tag(config[:container], output, 'style' => css) if @css
+          if instance_variable_defined?("@css")
+            output.replace content_tag(config[:container], output, 'style' => css) if @css
```

Comment:
Better to just change the condition at the end to `if instance_variable_defined?('@css')`